optimizely/helpers/audience.py

```diff
@@ -15,29 +15,39 @@
 
 from . import condition as condition_helper
 from . import condition_tree_evaluator
-from .enums import AudienceEvaluationLogs as audience_logs
+from .enums import ExperimentAudienceEvaluationLogs as experiment_audience_logs
+from .enums import RolloutRuleAudienceEvaluationLogs as rules_audience_logs
 
 
-def is_user_in_experiment(config, experiment, attributes, logger):
+def does_user_meet_audience_conditions(config,
+                                       audience_conditions,
+                                       experiment_or_rollout_rule,
+                                       logging_key,
+                                       attributes,
+                                       logger):
     """ Determine for given experiment if user satisfies the audiences for the experiment.
 
-  Args:
-    config: project_config.ProjectConfig object representing the project.
-    experiment: Object representing the experiment.
-    attributes: Dict representing user attributes which will be used in determining
-                if the audience conditions are met. If not provided, default to an empty dict.
-    logger: Provides a logger to send log messages to.
+    Args:
+        config: project_config.ProjectConfig object representing the project.
+        audience_conditions: Audience conditions corresponding to the experiment or rollout rule.
+        experiment_or_rollout_rule: String representing whether entity being evaluated is experiment or rollout rule.
```

Comment:
I think this could be cleaner if:
- `audience_logs` was an argument, rather than `experiment_or_rollout_rule`
- The association between rollout rule/experiment and the corresponding logs was abstracted by the `Experiment` entity class, and internally represented by a dict, or class property/method (maybe make rollout rule its own entity class?)

Reply:
I will adopt the `audience_logs` change now.

> maybe make rollout rule its own entity class

I was thinking about that, but in light of more incoming product changes where this terminology will change a lot, I am hoping that we make the transition then. Let me know if that sounds ok.